.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
Don't know if we want to add this file. There are many different editors out there, and they each have their own config file. WebTorrent Desktop is built from many npm packages, each with their own git repo.

So I don't think we want to add vim/emacs/intellij/etc configs for every editor to every repo. I recommend putting those in your home folder